package.json

```diff
@@ -121,6 +120,7 @@
 		"requireindex": "^1.1.0",
 		"rimraf": "^2.6.1",
 		"rxjs": "^6.6.7",
+		"sass": "^1.77.6",
```

Comment:
our recommendation is to [pin dependencies to a patch](https://github.com/guardian/recommendations/blob/main/dependencies.md#javascript), rather than using ranges.

Reply:
I shall pin the SASS relevant dependencies